pkgs/development/compilers/zig/0.9/setup-hook.sh

```diff
@@ -1,5 +1,6 @@
 # shellcheck shell=bash disable=SC2154,SC2086
```

Comment:
## [SC2154](https://www.shellcheck.net/wiki/SC2154) var is referenced but not assigned.

What triggers this lint?

## [SC2086](https://www.shellcheck.net/wiki/SC2086) Double quote to prevent globbing and word splitting.

What triggers this lint?